CodeEdit/NavigatorSidebar/FindNavigator/FindNavigator.swift

```diff
@@ -31,10 +35,45 @@ struct FindNavigator: View {
     var body: some View {
         VStack {
             VStack {
-                FindNavigatorModeSelector()
+                FindNavigatorModeSelector(state: state)
                 FindNavigatorSearchBar(state: state, title: "", text: $searchText)
                 HStack {
+                    Button {} label: {
+                        Text("In Workspace")
+                            .font(.system(size: 10))
+                    }.buttonStyle(.borderless)
                     Spacer()
+                    Menu {
+                        Button {
+                            currentFilter = filters[0]
+                            state.ignoreCase = true
+                            state.search(searchText)
+                        } label: {
+                            Text(filters[0])
+                        }
+                        Button {
+                            currentFilter = filters[1]
+                            state.ignoreCase = true
```

Comment:
I think this is supposed to be `state.ignoreCase = false`.

Reply:
Fixed